contributors/devel/automation.md

```diff
@@ -6,65 +6,26 @@ Kubernetes uses a variety of automated tools in an attempt to relieve developers
 of repetitive, low brain power work. This document attempts to describe these
 processes.
 
+## Tide
 
-## Submit Queue
+This project formerly used a Submit Queue, it has since been replaced by
+[Tide](https://git.k8s.io/test-infra/prow/cmd/tide).
 
-In an effort to
-   * reduce load on core developers
-   * maintain end-to-end test stability
-   * load test github's label feature
-
-We have added an automated [submit-queue](https://git.k8s.io/test-infra/mungegithub/submit-queue)
-to the
-[github "munger"](https://git.k8s.io/test-infra/mungegithub)
-for kubernetes.
-
-The submit-queue does the following:
-
-```go
-for _, pr := range readyToMergePRs() {
-    if testsAreStable() {
-        if retestPR(pr) == success {
-            mergePR(pr)
-        }
-    }
-}
-```
-
-The status of the submit-queue is [online.](http://submit-queue.k8s.io/)
-
-### Ready to merge status
+#### Ready to merge status
 
 A PR is considered "ready for merging" by the submit queue if it matches the set
 of conditions listed in the [merge requirements tab](http://submit-queue.k8s.io/#/info)
```

Comment:
Update this link to point to the tide dashboard. https://prow.k8s.io/tide

Reply:
Done.